finetuner/data.py

```diff
@@ -58,6 +63,28 @@ def build_dataset(
     return data
 
 
+def build_encoding_dataset(
+    model: Union['InferenceEngine', str], data: Union[List[str], DocumentArray]
+) -> DocumentArray:
+    """If data has been provided as a list, a :class:`DocumentArray` is created
+    from the elements of the list
+    """
```

Comment:
The parameter descriptions are missing

Reply:
@gmastrapas suggested they be removed as this is an internal function.